modules/serverless-configuring-kube-rbac-proxy-resources-for-serving.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * knative-serving/kube-rbac-proxy-serving.adoc
+
+:_content-type: REFERENCE
+[id="serverless-configuring-kube-rbac-proxy-resources-for-serving_{context}"]
+= Configuring kube-rbac-proxy resources for Serving
+
+You can globally override resource allocation for the `kube-rbac-proxy` container by using the {ServerlessOperatorName} CR.
```

Comment:
We also could mention that user can change that on a per deployment basis (we have a per deployment configuration docs elsewhere) but I feel that users just want to set this globally. It shouldn't be different for each deployment.

Reply:
@skonto We must have some description, so I went with this:
`The kube-rbac-proxy component provides internal authentication and authorization capabilities for Knative Eventing.`

Reply:
(nvm previous comment)
Added the notes.